app/src/main/java/org/android/go/sopt/data/remote/service/ListUsersService.kt

```diff
@@ -0,0 +1,10 @@
+package org.android.go.sopt.data.remote.service
+
+import org.android.go.sopt.data.remote.model.ResponseListUsersDto
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface ListUsersService {
+    @GET("/api/users?page=2")
```

Comment:
이러한 쿼리문 형태의 api는 `@Query`이용해서 메소드 작성해주는게 가독성 좋답니당  아무래도 url이 길어지면 보기가 힘들어지기 때문에 어노테이션을 이용해서 작성해주는게 좋습니다 !
```suggestion
    @GET("/api/users")
   fun getListUsers(
        @Query("page") num: Int = 2
    ): Call<ResponseListUsersDto>
```

Reply:
우왕 꿀팁 감사합니다 ㅎㅎ